backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/BpkTextField.kt

```diff
@@ -104,123 +60,3 @@ fun BpkTextField(
         interactionSource = interactionSource,
     )
 }
-
-@OptIn(ExperimentalAnimationApi::class)
-@Composable
-fun BpkTextField(
```

Comment:
this is a public API, so we can't just remove it unfortunately - what's the reason for removing it?

Reply:
Sorry, will get it back.